frontend/app.tsx

```diff
@@ -255,21 +266,35 @@ function reducer(
       if (action.filters.equals(state.filters)) {
         return state;
       }
-      const allIssues = [...state.allIssuesMap.values()];
+      // TODO (mlaw): re-hydrate stream from past events
```

Comment:
I don't understand what this todo means, can you expand slightly?

Reply:
This is fixed in another branch but when creating new operations and attaching them to an existing source, the source doesn't fire any events.

In other words, after changing filters we end up with 0 tasks. The same idea as your `experimentalWatch` API and `initialValuesInFirstDiff`